waspc/data/Generator/templates/react-app/src/queryClient.js

```diff
@@ -1,3 +1,21 @@
 import { QueryClient } from 'react-query'
 
-export const queryClient = new QueryClient()
+let queryClientConfig, resolveQueryClientInitialized, isQueryClientInitialized
+
+export const queryClientInitialized = new Promise(resolve => {
+  resolveQueryClientInitialized = resolve
+});
+
+export function configureQueryClient(config) {
+  if (isQueryClientInitialized) {
+    throw new Error("Attempted to configure the QueryClient after initialization")
```

Comment:
Can we recommend them how to fix it, maybe? Or would we be assuming too much at this point?

Reply:
filip said no it would be a bit messy